x/ibc/20-transfer/types/packet.go

```diff
@@ -5,28 +5,34 @@ import (
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	channelexported "github.com/cosmos/cosmos-sdk/x/ibc/04-channel/exported"
 )
 
-// PacketData defines a struct for the packet payload
-type PacketData struct {
+var _ channelexported.PacketDataI = PacketDataTransfer{}
```

Comment:
Why are these the same interface? This is a whole packet, e.g. with data, but we defined `PacketDataI` as the commitment I think (another confusing name).

Reply:
I suppose it's OK if packets *implement* this interface, but it's a bit confusing.